local/jimm/qa-lxd-mac.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# This script sets up a JIMM service within docker compose on macOS and adds an LXD controller.
+# It uses the default branch of jimm for the deployment.
+#
+# It requires the following tools:
+# - multipass
+# - go
+# - jq
+
+VM_NAME="jimm"
+
+echo "Setting up env."
+vm_exists=$(multipass list --format json | jq -r ".list[] | select(.name == \"$VM_NAME\") | .name")
+if [ -n "$vm_exists" ]; then
+  echo "Please delete $VM_NAME and try again."
+  exit 1
+fi
+
+# Cleaning ./tmp. Permission issues can arise when mounting ./tmp.
+sudo rm -rf ../../tmp
+
+echo "VM does not exist, launching VM: $VM_NAME"
+multipass launch --cpus 4 docker -n $VM_NAME
+
+# Multipass detects the vm name is the same as parent working path dir and creates a fuse mount.
+# We want a classic mount to reflect changes on the host to the VM.
+echo "Setting up classic mount"
+multipass umount jimm jimm || true
+multipass mount --type=classic ../../ jimm:jimm || true 
```

Comment:
Why the || true? If it fails wouldn't the rest of the script fail to work?

Reply:
I want the script to continue even if this fails